src/components/CountdownPane/CountdownPane.js

```diff
@@ -1,32 +1,84 @@
 import "./CountdownPane.css"
 import sunface from "../../assets/sunface.png"
+import React, { useEffect, useState } from "react";
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+export const CountdownPane = ({est, err}) => {
+  const [etd, setEtd] = useState(est)
+  const [timeLeft, setTimeLeft] = useState(0)
+  const [percentage, setPercentage] = useState(0)
+  const [error] = useState(err)
+
+  useEffect(() => {
+    calculateTimeLeft()
+  }, [])
+
+  const calculateTimeLeft = () => {
+    let newYear, newEtd
+    const splitEtd = etd.split("/")
+    if (splitEtd[2].length < 4) {
+      newYear = Number(`21${splitEtd[2]}`) + 122
+      newEtd = `${splitEtd[0]}/${splitEtd[1]}/${newYear}`
+      setEtd(newEtd)
+      calculateProgress()
+    }
+
+    const difference = +new Date(`${newEtd}`) - +new Date()
+    let updatedTimeLeft = {}
+  
+    if (difference > 0) {
+      updatedTimeLeft = {
+        days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+        seconds: Math.floor((difference / 1000) % 60)
+      }
+    }
+    setTimeLeft(updatedTimeLeft)
+  }
+
+  const calculateProgress = () => {
+    const total = +new Date(`${etd}`)
```

Comment:
Can this + operator be used in place of parseInt in other situations?

Reply:
I'm not sure! It seems to be date specific.